pkg/render/fluentd.go

```diff
@@ -274,7 +281,7 @@ func (c *fluentdComponent) Objects() ([]client.Object, []client.Object) {
 	if c.cfg.Filters != nil {
 		objs = append(objs, c.filtersConfigMap())
 	}
-	if c.cfg.EKSConfig != nil && c.cfg.OSType == rmeta.OSTypeLinux {
+	if c.cfg.EKSConfig != nil && c.cfg.OSType == rmeta.OSTypeLinux && c.cfg.EnterpriseCRDExists {
```

Comment:
I don't think we need Enterprise or not in this case.

Reply:
For OSS, EKS cloud watch is not needed. Hence I think this check is needed.